### PR TITLE
JBDS-3685 enable self-extracting AND running installation from archive

### PR DESCRIPTION
**Pull request by nickboldt**

### CODE REVIEW

**Comment by ppitonak** [comment]
Can one of the admins verify this patch?

File: config.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+;!@Install@!UTF-8!
+Title="Red Hat Developer Platform"
+BeginPrompt="Do you want to install the Red Hat Developer Platform?"
```
Comment (dgolovin) [inline config.txt]: There should be no additional prompt, Pete asked to remove that after first demo.

Comment (nickboldt) [inline config.txt]: PR updated. No more prompt.

**Comment by ppitonak** [comment]
add to whitelist

**Comment by ppitonak** [comment]
@nickboldt is this PR still valid?

**Comment by nickboldt** [comment]
A variation on this was already merged last week, so closing.

**Pull request closed without merging**